updater/dart_bindings/lib/src/bindings.dart

```diff
@@ -38,7 +41,13 @@ class AppParameters extends ffi.Struct {
     if (updateUrl != null) {
       config.ref.update_url = updateUrl.toNativeUtf8();
     }
-    config.ref.original_libapp_path = libappPath.toNativeUtf8();
+    config.ref.original_libapp_paths = calloc<ffi.Pointer<Utf8>>(
+      libappPaths.length + 1, // +1 for the null terminator.
```

Comment:
You don't need the null terminator anymore with your new method of passing the size.

Reply:
oh yeah good point 👍

Reply:
Ah yeah good call.